tools/util/getDiff.ts

```diff
@@ -0,0 +1,43 @@
+import { execSync } from "child_process";
+import { basename, dirname } from "path";
+
+export type ValidEventName = "push" | "pull_request" | "uncommitted";
```

Comment:
`uncommitted` is not valid

Reply:
I copied that from the previous implementation, I don't think this is used / needed, shall I remove?

Reply:
Yes

Reply:
Removed